src/server/server.js

```diff
@@ -104,7 +105,9 @@ app.use('/*', (req, res, next) => {
         }
     }
 });
-
+const redisClient = redisstorage.init({ config: config.redis });
+app.use(sessionStore(config, redisClient));
```

Comment:
Hører ikke denne mer hjemme der den lå, sammen med annen middleware? Virker også litt merkelig at `tildeling/storage` er ansvarlig for å sette opp Redis-klienten, det burde ligge utenfor en spesifikk komponent når den brukes av flere komponenter.

Reply:
Enig